hail/src/test/scala/is/hail/stats/LogisticRegressionModelSuite.scala

```diff
@@ -140,110 +140,6 @@ class LogisticRegressionModelSuite extends SparkSuite {
     assert(D_==(scoreStats.p, 0.3724319159, tolerance = 1.0E-5))
   }
 
-  @Test def covariatesVsInterceptOnlyR() {
```

Comment:
can you make an issue to add this test back in Python / Scala without R dependence and randomly assign? I concluded a few weeks ago that this is an important unit test

Reply:
I agree. @cseed Can you make an issue?

Reply:
Done, thanks: https://github.com/hail-is/hail/issues/6080